tests/build/chains.go

```diff
@@ -153,26 +152,19 @@ var _ = framework.ChainsSuiteDescribe("Tekton Chains E2E tests", Label("ec"), fu
 				publicKey, err := kubeController.GetPublicKey("signing-secrets", "tekton-chains")
 				Expect(err).ToNot(HaveOccurred())
 				GinkgoWriter.Println("Copy public key from tekton-chains/signing-secrets to a new secret")
-				Expect(kubeController.CreateOrUpdateSigningSecret(
-					publicKey, publicSecretName, namespace)).To(Succeed())
+				Expect(kubeController.CreateOrUpdateSigningSecret(publicKey, publicSecretName, namespace)).To(Succeed())
 
 				rekorHost, err = kubeController.GetRekorHost()
 				Expect(err).ToNot(HaveOccurred())
 				GinkgoWriter.Printf("Configured Rekor host: %s\n", rekorHost)
+
+				// make sure that the buildah pipeline with the produced the
+				// required images
+				err = kubeController.AwaitAttestationAndSignature(imageWithDigest, attestationTimeout)
+				Expect(err).NotTo(HaveOccurred())
 			})
 
 			BeforeEach(func() {
-				generator = tekton.VerifyEnterpriseContract{
```

Comment:
I don't think we should change this. ginkgo will ensure isolation between tests. From the [docs](https://onsi.github.io/ginkgo/#extracting-common-setup-beforeeach):

> each spec has a pristine, correctly initialized, copy of the shared variable.

Reply:
I thought I was seeing an issue with concurrent access to the same variable, though in retrospect that could be due to one of the issues I had with the Task definition. Easy to remove and it makes the test simpler...